test/src/graph/nodes/VelocityDistortRaysNodeTest.cpp

```diff
@@ -0,0 +1,103 @@
+#include <RGLFields.hpp>
+#include <graph/Node.hpp>
+#include <gtest/gtest.h>
+#include "utils.hpp"
+#include "scenes.hpp"
+
+using namespace ::testing;
+
+class VelocityDistortRaysNodeTest : public RGLTest, public RGLPointTestHelper {};
+
+TEST_F(VelocityDistortRaysNodeTest, invalid_arguments)
+{
+	rgl_node_t velocityDistortRaysNode;
+	rgl_vec3f velocity;
+	rgl_vec3f angularVelocity;
+
+	EXPECT_RGL_INVALID_ARGUMENT(rgl_node_rays_velocity_distort(nullptr, nullptr, nullptr), "node != nullptr");
+	EXPECT_RGL_INVALID_ARGUMENT(rgl_node_rays_velocity_distort(nullptr, &velocity, &angularVelocity), "node != nullptr");
+	EXPECT_RGL_INVALID_ARGUMENT(rgl_node_rays_velocity_distort(&velocityDistortRaysNode, nullptr, &angularVelocity), "velocity != nullptr");
+	EXPECT_RGL_INVALID_ARGUMENT(rgl_node_rays_velocity_distort(&velocityDistortRaysNode, &velocity, nullptr), "angularVelocity != nullptr");
+}
+
+TEST_F(VelocityDistortRaysNodeTest, valid_arguments)
+{
+	rgl_node_t velocityDistortRaysNode = nullptr;
+	rgl_vec3f velocity = { 1.0f, 1.0f, 1.0f };
+	rgl_vec3f angularVelocity = { 0.0f, 0.0f, 1.0f };
+
+	EXPECT_RGL_SUCCESS(rgl_node_rays_velocity_distort(&velocityDistortRaysNode, &velocity, &angularVelocity));
+	ASSERT_THAT(velocityDistortRaysNode, testing::NotNull());
+
+	// If (*node) != nullptr
+	EXPECT_RGL_SUCCESS(rgl_node_rays_velocity_distort(&velocityDistortRaysNode, &velocity, &angularVelocity));
+}
+
+TEST_F(VelocityDistortRaysNodeTest, use_case)
```

Comment:
Lack of tests:
- angular velocity
- linear and angular velocities combined
- pipeline with transform node (before / after distortion)
- distance field calculation
It is OK if these tests would be added in another PR.